test/TEST-48-UDEV-EVENT-TIMEOUT/test.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+set -e
+TEST_DESCRIPTION="test udev's event-timeout and timeout-signal options"
```

Comment:
Okay, last change (pinky swear!): skip this test in nspawn since we can't run a separate udev instance there.

```bash
TEST_NO_NSPAWN=1
```

should do the trick. With this change everything should finally pass.